manifests/testing/ocp/readme.txt

```diff
@@ -0,0 +1 @@
+place manifests for ocp here, ending in *for-ocp-yaml.in
```

Comment:
it is already in ocp/ directory, why bother with the suffix?

Reply:
it is used in testing code, which iterates all manifests in testing and subdirectories